rmd/Affixes.json

```diff
@@ -3266,18 +3266,18 @@
   },
   "820256": {
     "OriginalText": "ダメージを受けたとき\n１５％の確率で玄け反らなくなる。\n[a 03]※同名の特殊能力は重復しない[b]",
-    "Text": "",
-    "Enabled": false
+    "Text": "When receiving damage there is a\n15% chance nullifying hate accumulation.\n[a 03]＊Abilities of the same name cannot be\nstacked.[b]",
+    "Enabled": true
```

Comment:
This and others below it may not fit due to space requirements.

Reply:
I apologize, I forgot about the restriction.
Just 1 question, since the text is limited to 40 characters per line, "/n" is considered two characters? Does this mean that the text should be limited to 38 characters + "/n"? or can i write 40 characters +"/n"?

Reply:
`\n` is counted as a singular line break and does not count against the text limit.

Reply:
Okay, thx

Reply:
Fixed